client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -239,9 +244,12 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 				);
 			}
 			setPendingAction( () =>
-				setDesignOnSite( siteSlugOrId, _selectedDesign, siteVerticalId ).then( () =>
-					reduxDispatch( requestActiveTheme( site?.ID || -1 ) )
-				)
+				setDesignOnSite(
+					siteSlugOrId,
+					_selectedDesign,
+					selectedStyleVariation,
+					siteVerticalId
+				).then( () => reduxDispatch( requestActiveTheme( site?.ID || -1 ) ) )
 			);
 			recordTracksEvent( 'calypso_signup_select_design', {
 				...getEventPropsByDesign( _selectedDesign ),
```

Comment:
Slightly related, can we also track add the selected variation in `getEventPropsByDesign`?

Reply:
This is tracked in 131-gh-Automattic/ganon-issues. I think there was some (unconclusive?) discussion on how to track selected theme/design with style variation. Let's discuss about this later!

Reply:
Sure!